schema/cassandra/temporal/schema.cql

```diff
@@ -94,6 +94,22 @@ CREATE TABLE tasks (
     'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
   };
 
+-- Stores task queue information such as user provided versioning data
+CREATE TABLE task_queue_user_data (
+  namespace_id    uuid,
+  task_queue_name text,
+  data            binary, -- temporal.server.api.persistence.v1.TaskQueueUserData
+  data_encoding   text,   -- Encoding type used for serialization, in practice this should always be proto3
+  version         int64,  -- Version of this row, used for optimistic concurrency
+  -- task_queue_name is not a part of the parititioning key to allow iterating all task queues in a single namespace.
```

Comment:
nitpick: even if it was part of the partition key, we could still iterate. it would just be more expensive
```suggestion
  -- task_queue_name is not a part of the parititioning key to allow cheaply iterating all task queues in a single namespace.
```